### ISSUE TITLE
Warn during build process

### ISSUE THREAD

**Issue by vvadymk** [nuxt-modules/supabase#282]
I get this warn during the build process
` WARN  Export "serverSupabaseClient" of module "node_modules/.pnpm/@nuxtjs+supabase@1.1.0/node_modules/@nuxtjs/supabase/dist/runtime/server/services/serverSupabaseClient.mjs" was reexported through module "node_modules/.pnpm/@nuxtjs+supabase@1.1.0/node_modules/@nuxtjs/supabase/dist/runtime/server/services/index.mjs" while both modules are dependencies of each other and will end up in different chunks by current Rollup settings. This scenario is not well supported at the moment as it will produce a circular dependency between chunks and will likely lead to broken execution order.
Either change the import in "server/api/openAI/send.post.ts" to point directly to the exporting module or reconfigure "output.manualChunks" to ensure these modules end up in the same chunk.
`
Does it matter any sense?

**Comment by marco-quintella** [comment]
Getting same warning, but no error regarding it catch yet.

**Comment by sytexa-julia** [comment]
We also get this warning, apparently for each usage in server routes and apis
 @nuxtjs/supabase 1.3.2

**Comment by WindOfXaos** [comment]
Also getting these warnings @nuxtjs/supabase: 1.3.5

**Comment by jontybrook** [comment]
Also seeing this warning in build logs. No obvious impact in production at this time.

**Comment by th1m0** [comment]
Does this warning also occur in `v1.4.1`? If it does, could you provide a reproduction repo or StackBlitz?

**Comment by nicokempe** [comment]
> Does this warning also occur in `v1.4.1`? If it does, could you provide a reproduction repo or StackBlitz?

Yes, this warning still occurs on the latest version I tested: `@nuxtjs/supabase@2.0.4`.

I can reproduce it consistently across multiple projects and Nitro presets (including `cloudflare-module`, `bun`, and `node`).

In my experience, running `nuxi build` is enough to trigger the warning as soon as Nuxt Supabase server composables/helpers are used (for example `serverSupabaseClient`).

**Comment by v0develop** [comment]
Hi @larbish, is there some way we could support the resolution of this issue? Do you have an idea what could cause this issue and how we could fix this?
